apps/docs/content/components/DuotoneText/react.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: DuotoneText
+status: Experimental
+source: https://github.com/primer/brand/tree/main/packages/react/src/DuotoneText/DuotoneText.tsx
+storybook: '/brand/storybook/?path=/story/components-duotonetext--default'
+description: Use the duotone text component to style text with an emphasized portion.
+a11yReviewed: false
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+```js
+import {DuotoneText} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<Text>
+  <DuotoneText>
+    <DuotoneText.Emphasis>GitHub Codespaces</DuotoneText.Emphasis> offers a
+    complete dev environment in seconds.
+  </DuotoneText>
+</Text>
+```
+
+### With Bento
+
+```jsx live
+<Bento>
+  <Bento.Item rowSpan={5} flow="column">
+    <Bento.Content>
+      <Bento.Heading size="3">
+        <DuotoneText>
+          <DuotoneText.Emphasis>This is my super-sweet</DuotoneText.Emphasis>{' '}
+          bento heading
+        </DuotoneText>
+      </Bento.Heading>
+      <Link href="#">Call to action</Link>
+    </Bento.Content>
+    <Bento.Visual position="50% 100%">
+      <img
+        alt="placeholder, blank area with an gray background color"
+        src="https://via.placeholder.com/600x400/f5f5f5/f5f5f5.png"
+      />
+    </Bento.Visual>
+  </Bento.Item>
+</Bento>
+```
+
+### With RiverBreakout
+
+```jsx live
+<RiverBreakout>
+  <RiverBreakout.A11yHeading>
+    River breakout duotone text
+  </RiverBreakout.A11yHeading>
+  <RiverBreakout.Visual>
+    <img
+      src="https://via.placeholder.com/600x400/f5f5f5/f5f5f5.png"
+      alt="placeholder, blank area with an off-white background color"
+    />
+  </RiverBreakout.Visual>
+  <RiverBreakout.Content>
+    <Text>
+      <DuotoneText>
+        <DuotoneText.Emphasis>Lorem ipsum dolor sit amet,</DuotoneText.Emphasis>{' '}
+        consectetur adipiscing elit. In sapien sit ullamcorper id. Aliquam
+        luctus sed turpis felis nam pulvinar risus elementum.
+      </DuotoneText>
+    </Text>
+    <Link href="#">Call to action</Link>
+  </RiverBreakout.Content>
```

Comment:
Does DuotoneText need to be a child of Text? Could we instead do something like this:

```suggestion
 <RiverBreakout.Content>
    <DuotoneText>
      <DuotoneText.Emphasis>Lorem ipsum dolor sit amet,</DuotoneText.Emphasis>{' '}
      consectetur adipiscing elit. In sapien sit ullamcorper id. Aliquam
      luctus sed turpis felis nam pulvinar risus elementum.
    </DuotoneText>
    <Link href="#">Call to action</Link>
  </RiverBreakout.Content>
```

This would make DuotoneText functionally equivalent to Text, which i'm not actually sure is a good thing, but it could cut down on the amount of code needed to make the pattern work.

I'm not 100% on this idea 😄 

Another idea:

```jsx
<Text>
  <Text.Emphasis>Lorem ipsum dolor sit amet,</Text.Emphasis>{" "}
   consectetur adipiscing elit. In sapien sit ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus elementum.
</Text>
```

But at that point, it's probably easier to just stick with a visually-oriented HTML child like `b` or `strong` 🤷